Games/core/Meeting.js

```diff
@@ -646,7 +657,7 @@ module.exports = class Meeting {
 
       if (highest.targets.length == 1) {
         //Winning vote
-        if (this.inputType == "boolean" && this.mustAct && this.includeNo) {
+        if (this.inputType == "boolean" && (this.mustAct || this.mustCondemn) && this.includeNo) {
```

Comment:
technically not necessary since village vote is not a boolean, but this is also ok